mobx_codegen/lib/src/template/method_override.dart

```diff
@@ -1,8 +1,9 @@
 import 'package:analyzer/dart/element/element.dart';
-import 'package:analyzer/dart/element/type.dart';
-import 'package:mobx_codegen/src/template/comma_list.dart';
-import 'package:mobx_codegen/src/template/params.dart';
-import 'package:mobx_codegen/src/template/util.dart';
+
+import '../type_names.dart';
```

Comment:
Oops, a tool I used relativized these. I can revert if you'd like.

Reply:
Ya let's keep as package imports for consistency

Reply:
Done